### ISSUE TITLE
Footer social media links - DAC_Usability_Links_Issue2

### ISSUE THREAD

**Issue by gtvj** [nationalarchives/ds-wagtail#187]
On page 58 of the report DAC identify a usability issue with ID `DAC_Usability_Links_Issue2` relating to the social media links in the footer.

DAC have recommended we set these links to have empty `alt` attributes. 

### Why was this not picked up in the in-house testing? 

It is not a WCAG failure.

**Comment by gtvj** [comment]
Let's:

- Introduce a heading (maybe as `sr-only`)
- Make the items a list
- Add visible text below the items
